File: javascript/pro_6.js
function solution(n) {
    var answer = 0;
    let three = [];
    while(n){
        three.push(n%3);
        n = parseInt(n/3);
    }
    let p=1;
    for(let i=three.length-1; i>=0; i--){
        answer+=(p*three[i]);
        p*=3;
    }
    return answer;
}

File: python/pro_6.py
def solution(A, B):
    answer = 0
    A.sort()
    B.sort()
    i,j = 0,0
    while(i!=len(A) and j!=len(B)):
        if A[i]<B[j]:
            i+=1
            j+=1
            answer+=1
        else:
            j+=1
    return answer